skeleton-with-sanity/cms/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "westlandsite",
+  "private": true,
+  "version": "1.0.0",
+  "description": "In this example we will be deploying a simple \"Hello World\" example with Next.js",
+  "main": "package.json",
+  "author": "Paraform <code@paraform.studio>",
+  "license": "UNLICENSED",
+  "scripts": {
+    "start": "sanity start",
+    "test": "sanity check",
+    "now-build": "sanity build"
+  },
+  "keywords": [
+    "sanity"
+  ],
+  "dependencies": {
+    "@sanity/base": "^0.140.17",
+    "@sanity/cli": "^0.140.17",
+    "@sanity/components": "^0.140.20",
+    "@sanity/core": "^0.140.20",
+    "@sanity/dashboard": "^0.140.19",
+    "@sanity/default-layout": "^0.140.20",
+    "@sanity/default-login": "^0.140.15",
+    "@sanity/desk-tool": "^0.140.20",
+    "bcp47": "^1.1.2",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "sanity-plugin-dashboard-widget-document-list": "^0.0.8",
+    "sanity-plugin-dashboard-widget-netlify": "^1.0.0"
+  },
+  "devDependencies": {
+    "babel-eslint": "^10.0.1",
+    "eslint": "^5.16.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-config-standard-react": "^7.0.2",
+    "eslint-plugin-import": "^2.17.2",
+    "eslint-plugin-node": "^9.0.1",
+    "eslint-plugin-promise": "^4.1.1",
+    "eslint-plugin-react": "^7.13.0",
+    "eslint-plugin-standard": "^4.0.0",
+    "prettier-eslint-cli": "^4.7.1"
+  }
```

Comment:
@paraform ☝️ is that ten dependencies added to the project to make sure I put a semi-colon after each code statement? 🤔

Reply:
... does that come with a sanity create project?